components/footer.jsx

```diff
@@ -17,8 +17,8 @@ var Footer = React.createClass({
             </div>
             <div className="col-xs-6">
               <ul className="list-unstyled">
-                <li><a href="https://www.mozilla.org/en-US/about/legal/terms/mozilla">Legal</a></li>
-                <li><a href="https://www.mozilla.org/en-US/privacy/websites">Privacy</a></li>
+                <li><a href="https://webmaker.org/terms">Legal</a></li>
+                <li><a href="https://www.mozilla.org/privacy/websites">Privacy</a></li>
```

Comment:
Spec says use `https://webmaker.org/en-US/terms ` but I think we shouldn't specify locale in the urls.